Cargo.toml

```diff
@@ -19,3 +19,6 @@ categories = [
 	"data-structures",
 ]
 keywords = ["blockchain", "avalanche", "subnets", "ash"]
+
+[patch.crates-io]
+ash_api = { path = "../hai/ash-api-rs" }
```

Comment:
`patch` is not appropriate here. We can just add a `path` field to the existing `ash_cli` dependency.

See https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies

Reply:
Yes we add that until https://github.com/AshAvalanche/ash-rs/commit/1ff639b010ff97c9ad8a844bd34773d63eceee0e

Should we just go back to this?